go.mod

```diff
@@ -2,12 +2,14 @@ module github.com/grafana/grafana-plugin-sdk-go
 
 go 1.14
 
+// At least version v1.3.0 creates issues with Grafana dependencies.
+replace github.com/grpc-ecosystem/go-grpc-middleware => github.com/grpc-ecosystem/go-grpc-middleware v1.2.2
```

Comment:
Is this a temporary till we find out a solution or a permanent replace? If it's permanent then I suppose a better issue description required here.

Reply:
Right now permanent :) Will try to add something more meaningful :+1:

Reply:
Think I figured out the problem is rather https://github.com/prometheus/common/releases/tag/v0.21.0 rather than go-grpc-middleware and will need to be addressed in Grafana instead.